app/javascript/components/Profile.js

```diff
@@ -0,0 +1,209 @@
+                <button
+                  className="flex-no-shrink bg-teal hover:bg-teal-dark border-teal hover:border-teal-dark text-sm border-4 text-white py-1 px-2 rounded"
+                  type="submit"
+                >
+                  Update Token
+                </button>
+              </div>
+            </form>
+          </div>
+        );
+      }}
+    </Mutation>
+  );
+};
+const UpdateGithubUsername = props => {
+  let input;
+  return (
+    <Mutation mutation={UPDATE_GITHUB_USERNAME}>
+      {(updateGithubUsername, { data, error, loading, called }) => {
+        if (error) {
+          return <div>{JSON.stringify(error)}</div>;
+        }
+        if (loading) {
+          return <div>Submitting...</div>;
+        }
+        if (called) {
+          return <div>Github Username Updated!</div>;
+        }
+        return (
+          <div>
+            <form
+              className="w-full max-w-sm"
+              onSubmit={e => {
+                e.preventDefault();
+                updateGithubUsername({
+                  variables: { githubUsername: input.value }
+                });
+                input.value = "";
+              }}
+            >
+              <div className="flex items-center border-b border-b-2 border-teal py-2">
+                <input
+                  className="appearance-none bg-transparent border-none w-full text-grey-darker mr-3 py-1 px-2 leading-tight focus:outline-none"
+                  type="text"
+                  ref={node => {
+                    input = node;
+                  }}
+                  placeholder={props.githubUsername}
+                  aria-label="Github Username"
+                />
+                <button
+                  className="flex-no-shrink bg-teal hover:bg-teal-dark border-teal hover:border-teal-dark text-sm border-4 text-white py-1 px-2 rounded"
+                  type="submit"
+                >
+                  Update Github Username
+                </button>
+              </div>
+            </form>
+          </div>
+        );
+      }}
+    </Mutation>
+  );
+};
+const UpdateGithubOrg = props => {
+  let input;
+  return (
+    <Mutation mutation={UPDATE_GITHUB_ORG}>
+      {(updateGithubOrg, { data, error, loading, called }) => {
+        if (error) {
+          return <div>{JSON.stringify(error)}</div>;
+        }
+        if (loading) {
+          return <div>Submitting...</div>;
+        }
+        if (called) {
+          return <div>Github Org Updated!</div>;
+        }
+        return (
+          <div>
+            <form
+              className="w-full max-w-sm"
+              onSubmit={e => {
+                e.preventDefault();
+                updateGithubOrg({
+                  variables: { githubOrg: input.value }
+                });
+                input.value = "";
+              }}
+            >
+              <div className="flex items-center border-b border-b-2 border-teal py-2">
+                <input
```

Comment:
Can we make this a `select` of the current `Organization`s in our db, instead of a text input.  By doing so, this will be our first step towards assigning a user to a validated organization.  Reason being, without that we can't fetch their data from GitHub.

And then in a future iteration, I was thinking that we'd add the ability for them to choose from a list of all of their organizations, by leveraging [this service endpoint](https://github.com/oddballio/magnifier/blob/master/app/services/github/service.rb#L104-L116), and presenting the user a list of their organizations.